deps/thpool/thpool.c

```diff
@@ -439,6 +438,15 @@ static void redisearch_thpool_push_chain_verify_init_threads(
   redisearch_thpool_verify_init(thpool_p);
 }
 
+static void redisearch_thpool_push_chain(
```

Comment:
Consider reusing this in `redisearch_thpool_push_chain_verify_init_threads` as well